closed/openj9_version_info.h.in

```diff
@@ -25,7 +25,7 @@
 #define OPENJ9_VERSION_INFO_H
 
 #define J9COMPILER_VERSION_STRING "@COMPILER_VERSION_STRING@"
-#define J9PRODUCT_NAME            "@PRODUCT_NAME@"
+#define J9PRODUCT_NAME            "@PRODUCT_NAME@ @PRODUCT_SUFFIX@"
```

Comment:
If `PRODUCT_SUFFIX` is empty, I don't think we want the embedded space.

Reply:
I switched it to use RUNTIME_NAME, but turns out there is a default PRODUCT_SUFFIX.
https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/openj9/make/autoconf/version-numbers#L44

Reply:
We don't need to worry about an embedded space. If you configure with `--with-product-suffix=""` it works as if you said `--with-product-suffix="Runtime Environment"`. So your earlier macro should work:
```
#define J9PRODUCT_NAME            "@PRODUCT_NAME@ @PRODUCT_SUFFIX@"
```

Reply:
Updated.